### ISSUE TITLE
Review page: Note should be an alert (not plain text)

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#103622]
A11y reported a defect level 1, indicating this should be addressed prior to launch 

> Page: Review application
> Issue 6: Important text should be in warning notification not just plain text

![Image](https://github.com/user-attachments/assets/dd82f6bd-96dc-4537-b07e-c02b92a62dc7)

> Operating system and version: [Version]
> Assistive Tech used: Voiceover desktop and mobile
> Expected severity level: 1
> Description: Users need to be explicitly informed that proceeding past this step locks their ability to edit. Users who may have made mistakes should pause and review before submission. A standard note may be overlooked, but a warning-style message will catch users attention
> How to recreate:
> 
> 1. Sign in authenticated
> 2. Navigate through entire form to Review submission page
> 3. Navigate to bottom of page and see note
> 4. Recommendation: Use “Warning notification” component to indicate importance but not critical failure
> 
> Page: Accredited representative authorizations
